layouts/authors/single.html

```diff
@@ -198,16 +216,45 @@ <h3 class="title-3 xs-mb-20">{{ .Site.Data.globals.enged_cta.title }}</h3>
   </div>
 </section>
 
-<script type="text/javascript">
-  const toggleToolTip = (el) => {
-    el.classList.toggle('tooltip');
-  }
+<script defer type="text/javascript">
   
-  const insightItems = document.getElementsByClassName("insight");
+    const toggleToolTip = (el) => {
+      el.classList.toggle('tooltip');
+    }
+    
+    const insightItems = document.getElementsByClassName("insight");
+
+    for (let i = 0; i < insightItems.length; i++) {
+      insightItems[i].addEventListener("touch", () => toggleToolTip(insightItems[i]))
+    }
+  document.addEventListener("DOMContentLoaded", async function() {
+    const getPageView = (url, currentUrl) => {
+      const urlRegex = /(http:\/\/localhost:1313\/)|(https:\/\/beta.section.io\/engineering-education\/)|(https:\/\/section.io\/engineering-education\/)/gi;
+      const trimmedUrl = url.replace(urlRegex, '');
+
+      return fetch(`https:\/\/api.www.section.io/pageviews/${trimmedUrl}`)
+        .then(res => res.json())
+        .then(data => {
+          if (data.pageViews) {
+            const numPageViews = parseInt(data.pageViews).toLocaleString();
+            document.querySelector(`#page-views-count-${currentUrl}`).innerText = numPageViews;
+          }
+        });
+    }
+
+    const applyPageViews = async (urls) => {
+      let currentUrl = 0;
+
+      while (currentUrl <= urls.length - 1) {
+        await getPageView(urls[currentUrl], currentUrl);
```

Comment:
Just remember you need a try/catch to handle await errors. You may not need to handle the error in your case.

Reply:
Errors are being handled at the previous level.